README.md

```diff
@@ -32,6 +32,10 @@ API.route({
   }
 })
 
+// this is equivalent to creating a create, read, update, and destroy
+// route. Options are folded into every route.
```

Comment:
Maybe worth mentioning here that passing `method` or `path` with mess things up?